### ISSUE TITLE
Good but needs some work like constant translation

### ISSUE THREAD

**Issue by ajgw** [woocommerce/woocommerce-gutenberg-products-block#5116]
## What do you like about the Cart & Checkout blocks?
The layout is excellent – I like it.

## What do you think is missing from the Cart & Checkout blocks?
Well with Blocks plugin 6.2.0 installed it is till very buggy: on checkout page edit I see labels in my own language which is german, but on frontend customers and I see them in native english. Editing the terms and conditions text is a wonderful idea. But editing is a bit frustrating and styles for paragraph like left, center or justified textalign not there. In my case – I am using Storefront theme from Woo – text is on page edit left but on frontend justified and the only paragraph on site which is justified.
I have this plugin active since few months and customers ordered with new cart and checkout and it seems anyone had any issues with checkout. Thanks.

**Comment by DarianDev** [comment]
hey this isse is known. Here: [#5005](https://github.com/woocommerce/woocommerce-gutenberg-products-block/issues/5005) 
hope that they fix it fast as possible! @nielslange already posted a suggestion

**Comment by DarianDev** [comment]
@nielslange

**Comment by nielslange** [comment]
Hello @ajgw 👋 Thanks for reporting this. 

As @DarianDev mentioned, the untranslated sections have been reported in #5005 and #5106, and I've created a fix for that in #5112.

Regarding aligning the terms and conditions text, I took a brief look and noticed that activating the alignment option for that section will lead to various layout problems. Given the current roadmap and priorities, I'm afraid this is not something we'll look into in the near future. However, you can control the alignment of that section using custom CSS.

The following snippet allows to right align the terms and conditions text:
```css
/* Right align terms & conditions text */
.wc-block-checkout__terms {
  text-align: right;
}
```

The following snippet allows centering the terms and conditions text:
```css
/* Center terms & conditions text */
.wc-block-checkout__terms {
  text-align: center;
}
```

You can add the desired CSS snippet to your site as described in https://www.wpbeginner.com/plugins/how-to-easily-add-custom-css-to-your-wordpress-site/.
